docs/reference/ingest/processors/data-stream-router.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[data-stream-router-processor]]
+=== Data stream router processor
+++++
+<titleabbrev>Data stream router</titleabbrev>
+++++
+
+The `data_stream_router` processor allows to route a document from one data stream to another data stream.
+It can use both static values or values from the document to determine the target data stream.
+
+The name of a data stream is comprised of three parts and looks like this: `<type>-<dataset>-<namespace>`.
+See the {fleet-guide}/data-streams.html#data-streams-naming-scheme[data stream naming scheme] documentation for more details.
+
+NOTE: `data_stream_router` processor can only be used on data streams that follow the data streams naming scheme.
+Trying to use this processor on a data stream with a non-compliant name will raise an exception.
+
+After a `data_stream_router` processor has been executed, all the other processors of the current pipeline are skipped.
+This means that at most one `data_stream_router` processor is ever executed within a pipeline,
+allowing to define mutually exclusive routing conditions,
+similar to a if, else-if, else-if, … condition.
+
```

Comment:
It might be worth mentioning that "caller" must have permissions to write to the final target data stream.

Reply:
done